avocado/core/nrunner.py

```diff
@@ -360,6 +360,10 @@ class BaseRunner(abc.ABC):
     Base interface for a Runner
     """
 
+    #: The "main Avocado" configuration keys (AKA namespaces) that
+    #: this runners makes use of.
+    CONFIGURATION_USED = []
```

Comment:
I think we should have ability to set just namespace here without listing all the configuration values from one namespace. For example will need all values from `core` namespace. Will be nice to do just this:
```
CONFIGURATION_USED = ['core']
```
instead of: 
```
CONFIGURATION_USED = ['core.input_encoding', 'core.paginator`, 'core.show', 'core.verbose']
```

Reply:
I have mixed feelings about this... it'd simplify the declaration of configuration indeed, but at the same time, depending on the namespace, it could considerably defeat the original purpose of this PR which is to bring clarity to the command line and make it easier to be reproduced.